process/sandbox.md

```diff
@@ -0,0 +1,99 @@
+* Remove possible legal and governance obstacles to adoption and contribution by ensuring all projects adhere to CNCF legal, code of conduct and IP Policy requirements
+
+This proposal is to move all Inception level projects to the Sandbox and provide clarity on what Sandbox projects stand for.
+
+## What is the CNCF Sandbox
+
+There’s been a desire within the CNCF TOC and community to provide further clarity around project maturity levels in CNCF.
+
+When we initially created the Inception project level, it was intended to provide an avenue for technically interesting early-stage projects that were beneficial to the cloud-native community.  We are transitioning Inception projects to the Sandbox. Sandbox projects should be early-stage projects that the CNCF TOC believes warrant experimentation. The Sandbox should provide a beneficial, neutral home for such projects, in order to foster collaborative development. We aspire to make the Sandbox the preferred path for early-stage projects to enter the CNCF. More mature projects can continue to jump directly to incubation, but as the cloud-native ecosystem grows, we expect to see proportionally more early-stage projects.
+
+## Early Stage
+
+When we say that Sandbox projects are "early stage" this covers the following examples:
+
+1. New projects that are designed to extend one or more CNCF projects with functionality or interoperability libraries. In the case of Kubernetes, the Sandbox is intended as a home for projects that would previously have [started in the Kubernetes Incubator](https://github.com/kubernetes/community/blob/master/incubator.md).
+2. Independent projects that fit the CNCF mission and provide potential for a novel approach to existing functional areas (or are an attempt to meet an unfulfilled need)
+3. Projects commissioned or sanctioned by the CNCF, including initial code for CNCF WG collaborations, and "experimental" projects
+4. Any project that realistically intends to join CNCF Incubation in future and wishes to lay the foundations for that
+
+## Roadmap for Sandbox Projects
+
+![CNCF Sandbox Roadmap](sandbox.png)
+
+## Caveat Utilitor
+
+The CNCF hopes that all early stage projects will achieve the success they desire.  And the organisation will help as appropriate.  But certain caveats must be stated nonetheless.
+
+End users should treat early stage projects with care.  It is expected that some Sandbox projects may fail.  They may never move to the next maturity level. While many early projects are safe to try out, users must exercise their own judgment.  Some projects may be alpha quality software.  There is no guarantee of production readiness, users, or professional level support.  Where projects enjoy the public support of one or more professional software organisations, those may be seed stage. In short: The CNCF Operating Principle about "no kingmakers" is of special importance in the Sandbox.  
```

Comment:
I would suggest even stronger wording here, specifically addressing what the CNCF does and not do when evaluating projects for inclusion in the sandbox.  For example, I assume that we do not plan to do:

1. any form of technical or market due diligence on the project's current state.
2. any form of significant endorsement that the project's approach to solving a given problem area is necessarily an  approach recommended by the CNCF/TOC. 

There are various words in this document that blurr this fact a bit, and I think that we need to be crystal clear.

Some example of such words elsewhere in the document:

- "sanctioned by the CNCF"...
- "CNCF TOC believes warrant experimentation"
- "Require 2 TOC sponsors"

What exactly does it mean to "sanction", "sponsor" and "deem to warrant experimentation".   I don't want to suggest that we exhaustively define these terms in this document, but I do think that we all need to be clear in our minds that the barrier for inclusion is intended to be as low as reasonably possible, and be clear exactly what that barrier is.  If necessary, just remove the above inferrences that there is some vague, unspecified barrier.

Reply:
@quinton-hoole 2 TOC sponsors simply means when they propose a project they will have to have 2 folks in the sponsor line (see OPA as an example, they will have to grab one more person outside of Ken: https://github.com/cncf/toc/pull/71)

Reply:
@caniszczyk You haven't addressed any of my questions :-)